src/Runner.Worker/Container/ContainerHooks/ContainerHookManager.cs

```diff
@@ -71,10 +73,9 @@ public async Task PrepareJobAsync(IExecutionContext context, List<ContainerInfo>
 
             SaveHookState(context, response.State);
 
-            // TODO: figure out if we need ContainerRuntimePath for anything
-            // var configEnvFormat = "--format \"{{range .Config.Env}}{{println .}}{{end}}\"";
-            // var containerEnv = await _dockerManager.DockerInspect(executionContext, container.ContainerId, configEnvFormat);
-            // container.ContainerRuntimePath = DockerUtil.ParsePathFromConfigEnv(containerEnv);
+            var configEnvFormat = "--format \"{{range .Config.Env}}{{println .}}{{end}}\"";
+            var containerEnv = await _dockerManager.DockerInspect(context, jobContainer?.ContainerId, configEnvFormat);
```

Comment:
When using hooks, we can't expect there to be a valid docker installation, so `containerEnv` is something we must get from the response file. The hook should run docker inspect (or equivalent) and just write it in the response.

The other side of this issue is to actually write this into response.context on the hooks side


This is not part of the [ADR](https://github.com/github/c2c-actions-runtime/blob/thboop/containerhooks/adrs/1748-container-hooks-interface.md#prepare_job) afaik, @thboop do you agree with the approach?

Reply:
What is containerEnv used for?